### DISCUSSION TITLE
My node refuses to reboot - Failed to start Bitcoin daemon - pls help

### DISCUSSION THREAD

**Discussion by KanutoTakumi** [raspiblitz/raspiblitz#3954] [Q&A] [answered]
***************************************************************
* RASPIBLITZ DEBUG LOGS 
***************************************************************
blitzversion: 1.8.0
chainnetwork: bitcoin / main
 20:24:20 up 3 min,  2 users,  load average: 2.48, 1.37, 0.55

*** SETUPPHASE / BOOTSTRAP ***
see logs: cat /home/admin/raspiblitz.log
setupPhase--> done
state--> ready

*** BACKGROUNDSERVICE ***
to monitor Background service call: sudo journalctl -f -u background

*** BLOCKCHAIN (MAINNET) SYSTEMD STATUS ***
● bitcoind.service - Bitcoin daemon
     Loaded: loaded (/etc/systemd/system/bitcoind.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Tue 2023-04-25 20:22:39 BST; 1min 40s ago
    Process: 6855 ExecStartPre=/home/admin/config.scripts/blitz.systemd.sh log blockchain STARTED (code=exited, status=0/SUCCESS)
    Process: 6866 ExecStartPre=/bin/chgrp bitcoin /mnt/hdd/bitcoin (code=exited, status=0/SUCCESS)
    Process: 6869 ExecStart=/usr/local/bin/bitcoind -daemonwait -conf=/mnt/hdd/bitcoin/bitcoin.conf -datadir=/mnt/hdd/bitcoin -debuglogfile=/mnt/hdd/bitcoin/debug.log (code=exited, status=1/FAILURE)
        CPU: 709ms

Apr 25 20:22:39 raspberrypi systemd[1]: bitcoind.service: Failed with result 'exit-code'.
Apr 25 20:22:39 raspberrypi systemd[1]: Failed to start Bitcoin daemon.

*** LAST BLOCKCHAIN (MAINNET) ERROR LOGS ***
sudo journalctl -u bitcoind -b --no-pager -n8
-- Journal begins at Mon 2023-04-24 05:39:35 BST, ends at Tue 2023-04-25 20:24:20 BST. --
Apr 25 20:22:39 raspberrypi systemd[1]: bitcoind.service: Control process exited, code=exited, status=1/FAILURE
Apr 25 20:22:39 raspberrypi systemd[1]: bitcoind.service: Failed with result 'exit-code'.
Apr 25 20:22:39 raspberrypi systemd[1]: Failed to start Bitcoin daemon.
Apr 25 20:22:39 raspberrypi systemd[1]: bitcoind.service: Scheduled restart job, restart counter is at 5.
Apr 25 20:22:39 raspberrypi systemd[1]: Stopped Bitcoin daemon.
Apr 25 20:22:39 raspberrypi systemd[1]: bitcoind.service: Start request repeated too quickly.
Apr 25 20:22:39 raspberrypi systemd[1]: bitcoind.service: Failed with result 'exit-code'.
Apr 25 20:22:39 raspberrypi systemd[1]: Failed to start Bitcoin daemon.

*** LAST BLOCKCHAIN (MAINNET) 20 INFO LOGS ***
sudo tail -n 20 /mnt/hdd/bitcoin/debug.log
2023-04-25T19:22:38Z Config file arg: [test] rpcport="18332"
2023-04-25T19:22:38Z Command-line arg: conf="/mnt/hdd/bitcoin/bitcoin.conf"
2023-04-25T19:22:38Z Command-line arg: daemonwait=""
2023-04-25T19:22:38Z Command-line arg: datadir="/mnt/hdd/bitcoin"
2023-04-25T19:22:38Z Command-line arg: debuglogfile="/mnt/hdd/bitcoin/debug.log"
2023-04-25T19:22:38Z Using at most 40 automatic connections (1024 file descriptors available)
2023-04-25T19:22:38Z Using 16 MiB out of 32/2 requested for signature cache, able to store 524288 elements
2023-04-25T19:22:38Z Using 16 MiB out of 32/2 requested for script execution cache, able to store 524288 elements
2023-04-25T19:22:38Z Script verification uses 3 additional threads
2023-04-25T19:22:38Z Wallet disabled!
2023-04-25T19:22:38Z scheduler thread start
2023-04-25T19:22:38Z HTTP: creating work queue of depth 16
2023-04-25T19:22:38Z Config options rpcuser and rpcpassword will soon be deprecated. Locally-run instances may remove rpcuser to use cookie-based auth, or may be replaced with rpcauth. Please see share/rpcauth for rpcauth auth generation.
2023-04-25T19:22:38Z HTTP: starting 4 worker threads
2023-04-25T19:22:38Z Using /16 prefix for IP bucketing
2023-04-25T19:22:38Z init message: Loading P2P addresses…
2023-04-25T19:22:39Z Error: Invalid or corrupt peers.dat (Checksum mismatch, data corrupted). If you believe this is a bug, please report it to https://github.com/bitcoin/bitcoin/issues. As a workaround, you can move the file ("/mnt/hdd/bitcoin/peers.dat") out of the way (rename, move, or delete) to have a new one created on the next start.
2023-04-25T19:22:39Z Shutdown: In progress...
2023-04-25T19:22:39Z scheduler thread exit
2023-04-25T19:22:39Z Shutdown: done

*** LND (MAINNET) SYSTEMD STATUS ***
● lnd.service - Lightning Network Daemon
     Loaded: loaded (/etc/systemd/system/lnd.service; enabled; vendor preset: enabled)
     Active: inactive (dead)

Apr 25 20:22:34 raspberrypi systemd[1]: Dependency failed for Lightning Network Daemon.
Apr 25 20:22:34 raspberrypi systemd[1]: lnd.service: Job lnd.service/start failed with result '…ency'.
Hint: Some lines were ellipsized, use -l to show in full.

*** LAST LND (MAINNET) ERROR LOGS ***
sudo journalctl -u lnd -b --no-pager -n12
-- Journal begins at Mon 2023-04-24 05:39:35 BST, ends at Tue 2023-04-25 20:24:20 BST. --
Apr 25 20:22:34 raspberrypi systemd[1]: Dependency failed for Lightning Network Daemon.
Apr 25 20:22:34 raspberrypi systemd[1]: lnd.service: Job lnd.service/start failed with result 'dependency'.

*** LAST 30 LND (MAINNET) INFO LOGS ***
sudo tail -n 30 /mnt/hdd/lnd/logs/bitcoin/mainnet/lnd.log
tail: cannot open '/mnt/hdd/lnd/logs/bitcoin/mainnet/lnd.log' for reading: No such file or directory

*** CORE LIGHTNING (MAINNET) SYSTEMD STATUS ***
- not activated -

*** BLOCKCHAIN (TESTNET) SYSTEMD STATUS ***
- OFF by config -
*** LND (TESTNET) SYSTEMD STATUS ***
- OFF by config -

*** CORE LIGHTNING (TESTNET) SYSTEMD STATUS ***
- not activated -

*** BLOCKCHAIN (SIGNET) SYSTEMD STATUS ***
- OFF by config -
*** LND (SIGNET) SYSTEMD STATUS ***
- OFF by config -

*** CORE LIGHTNING (SIGNET) SYSTEMD STATUS ***
- not activated -

*** NGINX SYSTEMD STATUS ***
● nginx.service - A high performance web server and a reverse proxy server
     Loaded: loaded (/lib/systemd/system/nginx.service; enabled; vendor preset: enabled)
    Drop-In: /etc/systemd/system/nginx.service.d
             └─raspiblitz.conf
     Active: active (running) since Tue 2023-04-25 20:24:01 BST; 18s ago
       Docs: man:nginx(8)
    Process: 14877 ExecStartPre=/usr/sbin/nginx -t -q -g daemon on; master_process on; (code=exited, status=0/SUCCESS)
    Process: 14889 ExecStart=/usr/sbin/nginx -g daemon on; master_process on; (code=exited, status=0/SUCCESS)
   Main PID: 14891 (nginx)
      Tasks: 5 (limit: 4531)
        CPU: 226ms
     CGroup: /system.slice/nginx.service
             ├─14891 nginx: master process /usr/sbin/nginx -g daemon on; master_process on;
             ├─14892 nginx: worker process
             ├─14893 nginx: worker process
             ├─14894 nginx: worker process
             └─14895 nginx: worker process

Apr 25 20:24:01 raspberrypi systemd[1]: Starting A high performance web server and a reverse pr…ver...
Apr 25 20:24:01 raspberrypi systemd[1]: Started A high performance web server and a reverse pro…erver.
Hint: Some lines were ellipsized, use -l to show in full.

*** LAST NGINX LOGS ***
sudo journalctl -u nginx -b --no-pager -n20
-- Journal begins at Mon 2023-04-24 05:39:35 BST, ends at Tue 2023-04-25 20:24:20 BST. --
Apr 25 20:21:43 raspberrypi nginx[665]: nginx: configuration file /etc/nginx/nginx.conf test failed
Apr 25 20:21:43 raspberrypi systemd[1]: nginx.service: Control process exited, code=exited, status=1/FAILURE
Apr 25 20:21:43 raspberrypi systemd[1]: nginx.service: Failed with result 'exit-code'.
Apr 25 20:21:43 raspberrypi systemd[1]: Failed to start A high performance web server and a reverse proxy server.
Apr 25 20:22:58 raspberrypi systemd[1]: nginx.service: Scheduled restart job, restart counter is at 1.
Apr 25 20:22:58 raspberrypi systemd[1]: Stopped A high performance web server and a reverse proxy server.
Apr 25 20:22:58 raspberrypi systemd[1]: Starting A high performance web server and a reverse proxy server...
Apr 25 20:22:59 raspberrypi nginx[8589]: nginx: [emerg] bind() to 0.0.0.0:50002 failed (98: Address already in use)
Apr 25 20:22:59 raspberrypi nginx[8589]: nginx: [emerg] bind() to 0.0.0.0:50002 failed (98: Address already in use)
Apr 25 20:23:00 raspberrypi nginx[8589]: nginx: [emerg] bind() to 0.0.0.0:50002 failed (98: Address already in use)
Apr 25 20:23:00 raspberrypi nginx[8589]: nginx: [emerg] bind() to 0.0.0.0:50002 failed (98: Address already in use)
Apr 25 20:23:01 raspberrypi nginx[8589]: nginx: [emerg] bind() to 0.0.0.0:50002 failed (98: Address already in use)
Apr 25 20:23:01 raspberrypi nginx[8589]: nginx: [emerg] still could not bind()
Apr 25 20:23:01 raspberrypi systemd[1]: nginx.service: Control process exited, code=exited, status=1/FAILURE
Apr 25 20:23:01 raspberrypi systemd[1]: nginx.service: Failed with result 'exit-code'.
Apr 25 20:23:01 raspberrypi systemd[1]: Failed to start A high performance web server and a reverse proxy server.
Apr 25 20:24:01 raspberrypi systemd[1]: nginx.service: Scheduled restart job, restart counter is at 2.
Apr 25 20:24:01 raspberrypi systemd[1]: Stopped A high performance web server and a reverse proxy server.
Apr 25 20:24:01 raspberrypi systemd[1]: Starting A high performance web server and a reverse proxy server...
Apr 25 20:24:01 raspberrypi systemd[1]: Started A high performance web server and a reverse proxy server.
--> CHECK CONFIG: sudo nginx -t
nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
nginx: configuration file /etc/nginx/nginx.conf test is successful

*** BLITZAPI SYSTEMD STATUS ***
● blitzapi.service - BlitzBackendAPI
     Loaded: loaded (/etc/systemd/system/blitzapi.service; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2023-04-25 20:21:42 BST; 2min 37s ago
   Main PID: 833 (python)
      Tasks: 7 (limit: 4531)
        CPU: 6.199s
     CGroup: /system.slice/blitzapi.service
             └─833 /usr/bin/python -m uvicorn app.main:app --port 11111 --host=0.0.0.0 --root-path /a…

Apr 25 20:22:33 raspberrypi python[833]: ERROR:root:Unknown answer from Bitcoin Core. Reason: I… Error
Apr 25 20:22:37 raspberrypi python[833]: ERROR:root:Unknown answer from Bitcoin Core. Reason: I… Error
Hint: Some lines were ellipsized, use -l to show in full.

*** LAST BLITZAPI LOGS ***
sudo journalctl -u blitzapi -b --no-pager -n20
-- Journal begins at Mon 2023-04-24 05:39:35 BST, ends at Tue 2023-04-25 20:24:20 BST. --
Apr 25 20:21:50 raspberrypi python[833]: INFO:     Waiting for application startup.
Apr 25 20:21:50 raspberrypi python[833]: INFO:uvicorn.error:Waiting for application startup.
Apr 25 20:21:50 raspberrypi python[833]: INFO:     Application startup complete.
Apr 25 20:21:50 raspberrypi python[833]: INFO:uvicorn.error:Application startup complete.
Apr 25 20:21:50 raspberrypi python[833]: INFO:     Uvicorn running on http://0.0.0.0:11111 (Press CTRL+C to quit)
Apr 25 20:21:50 raspberrypi python[833]: INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:11111 (Press CTRL+C to quit)
Apr 25 20:21:50 raspberrypi python[833]: WARNING:root:Key 'internet_online' not found in Redis DB.
Apr 25 20:21:50 raspberrypi python[833]: WARNING:root:Key 'hdd_capacity_bytes' not found in Redis DB.
Apr 25 20:21:50 raspberrypi python[833]: ERROR:asyncio:Task exception was never retrieved
Apr 25 20:21:50 raspberrypi python[833]: future: <Task finished name='Task-8' coro=<_handle_gather_hardware_info() done, defined at ./app/repositories/system.py:185> exception=ValueError("invalid literal for int() with base 10: ''")>
Apr 25 20:21:50 raspberrypi python[833]: Traceback (most recent call last):
Apr 25 20:21:50 raspberrypi python[833]:   File "./app/repositories/system.py", line 188, in _handle_gather_hardware_info
Apr 25 20:21:50 raspberrypi python[833]:     info = await get_hardware_info()
Apr 25 20:21:50 raspberrypi python[833]:   File "./app/repositories/system.py", line 132, in get_hardware_info
Apr 25 20:21:50 raspberrypi python[833]:     return await get_hardware_info_impl()
Apr 25 20:21:50 raspberrypi python[833]:   File "./app/repositories/hardware_impl/raspiblitz.py", line 55, in get_hardware_info_impl
Apr 25 20:21:50 raspberrypi python[833]:     total = int(await redis_get("hdd_capacity_bytes"))
Apr 25 20:21:50 raspberrypi python[833]: ValueError: invalid literal for int() with base 10: ''
Apr 25 20:22:33 raspberrypi python[833]: ERROR:root:Unknown answer from Bitcoin Core. Reason: Internal Server Error
Apr 25 20:22:37 raspberrypi python[833]: ERROR:root:Unknown answer from Bitcoin Core. Reason: Internal Server Error

- TOUCHSCREEN is OFF by config
- Loop is OFF by config

*** LND-RTL ***
● RTL.service - RTL Webinterface
     Loaded: loaded (/etc/systemd/system/RTL.service; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2023-04-25 20:22:36 BST; 1min 44s ago
    Process: 6414 ExecStartPre=/home/admin/config.scripts/bonus.rtl.sh prestart lnd mainnet (code=exited, status=0/SUCCESS)
   Main PID: 6678 (node)
      Tasks: 11 (limit: 4531)
        CPU: 6.339s
     CGroup: /system.slice/RTL.service
             └─6678 /usr/bin/node /home/rtl/RTL/rtl

Apr 25 20:22:34 raspberrypi systemd[1]: Starting RTL Webinterface...
Apr 25 20:22:36 raspberrypi systemd[1]: Started RTL Webinterface.

- CL-RTL is OFF by config

*** LAST 20 ElectRS LOGS ***
sudo journalctl -u electrs -b --no-pager -n20
-- Journal begins at Mon 2023-04-24 05:39:35 BST, ends at Tue 2023-04-25 20:24:20 BST. --
Apr 25 20:22:48 raspberrypi systemd[1]: electrs.service: Consumed 10.184s CPU time.
Apr 25 20:23:48 raspberrypi systemd[1]: electrs.service: Scheduled restart job, restart counter is at 1.
Apr 25 20:23:48 raspberrypi systemd[1]: Stopped Electrs.
Apr 25 20:23:48 raspberrypi systemd[1]: electrs.service: Consumed 10.184s CPU time.
Apr 25 20:23:48 raspberrypi systemd[1]: Started Electrs.
Apr 25 20:23:48 raspberrypi electrs[13275]: Starting electrs 0.9.9 on aarch64 linux with Config { network: Bitcoin, db_path: "/mnt/hdd/app-storage/electrs/db/bitcoin", daemon_dir: "/home/electrs/.bitcoin", daemon_auth: UserPass("raspibolt", "<sensitive>"), daemon_rpc_addr: 127.0.0.1:8332, daemon_p2p_addr: 127.0.0.1:8333, electrum_rpc_addr: 0.0.0.0:50001, monitoring_addr: 127.0.0.1:4224, wait_duration: 10s, jsonrpc_timeout: 15s, index_batch_size: 10, index_lookup_limit: None, reindex_last_blocks: 0, auto_reindex: true, ignore_mempool: false, sync_once: false, disable_electrum_rpc: false, server_banner: "Welcome to electrs v0.9.7 - the Electrum Rust Server on your RaspiBlitz", args: [] }
Apr 25 20:23:48 raspberrypi electrs[13275]: [2023-04-25T19:23:48.377Z INFO  electrs::metrics::metrics_impl] serving Prometheus metrics on 127.0.0.1:4224
Apr 25 20:23:48 raspberrypi electrs[13275]: [2023-04-25T19:23:48.377Z INFO  electrs::server] serving Electrum RPC on 0.0.0.0:50001
Apr 25 20:23:48 raspberrypi electrs[13275]: [2023-04-25T19:23:48.463Z INFO  electrs::db] "/mnt/hdd/app-storage/electrs/db/bitcoin": 165 SST files, 40.261766039 GB, 5.043417039 Grows
Apr 25 20:23:53 raspberrypi electrs[13275]: [2023-04-25T19:23:53.317Z INFO  electrs::chain] loading 786981 headers, tip=0000000000000000000029602197226079437cb9fd8ce5072c0c9757bb91d775
Apr 25 20:23:57 raspberrypi electrs[13275]: [2023-04-25T19:23:57.720Z INFO  electrs::chain] chain updated: tip=0000000000000000000029602197226079437cb9fd8ce5072c0c9757bb91d775, height=786981
Apr 25 20:23:57 raspberrypi electrs[13275]: [2023-04-25T19:23:57.749Z INFO  electrs::db] closing DB at /mnt/hdd/app-storage/electrs/db/bitcoin
Apr 25 20:23:57 raspberrypi electrs[13275]: Error: electrs failed
Apr 25 20:23:57 raspberrypi electrs[13275]: Caused by:
Apr 25 20:23:57 raspberrypi electrs[13275]:     0: bitcoind RPC polling failed
Apr 25 20:23:57 raspberrypi electrs[13275]:     1: daemon not available
Apr 25 20:23:57 raspberrypi electrs[13275]:     2: JSON-RPC error: transport error: Couldn't connect to host: Connection refused (os error 111)
Apr 25 20:23:57 raspberrypi systemd[1]: electrs.service: Main process exited, code=exited, status=1/FAILURE
Apr 25 20:23:57 raspberrypi systemd[1]: electrs.service: Failed with result 'exit-code'.
Apr 25 20:23:57 raspberrypi systemd[1]: electrs.service: Consumed 9.395s CPU time.

*** ElectRS Status ***
##### STATUS ELECTRS SERVICE
configured=1
serviceInstalled=1
serviceRunning=0
serviceRunning=0
tipSynced=0
initialSynced=0
electrumResponding=0
infoSync='Not running - check: sudo journalctl -u electrs'

*** LAST 20 LIT LOGS ***
sudo journalctl -u litd -b --no-pager -n20
-- Journal begins at Mon 2023-04-24 05:39:35 BST, ends at Tue 2023-04-25 20:24:21 BST. --
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.010 [DBG] GRPC: [core] Subchannel picks a new address "127.0.0.1:10009" to connect
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.017 [WRN] GRPC: [core] grpc: addrConn.createTransport failed to connect to {127.0.0.1:10009 127.0.0.1:10009 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:10009: connect: connection refused". Reconnecting...
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.017 [DBG] GRPC: [core] Subchannel Connectivity change to TRANSIENT_FAILURE
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.017 [DBG] GRPC: [core] pickfirstBalancer: UpdateSubConnState: 0x40005307a0, {TRANSIENT_FAILURE connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:10009: connect: connection refused"}
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.017 [DBG] GRPC: [core] Channel Connectivity change to TRANSIENT_FAILURE
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.017 [WRN] GRPC: [core] grpc: addrConn.createTransport failed to connect to {127.0.0.1:10009 127.0.0.1:10009 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:10009: connect: connection refused". Reconnecting...
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.018 [DBG] GRPC: [core] Subchannel Connectivity change to TRANSIENT_FAILURE
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.018 [DBG] GRPC: [core] pickfirstBalancer: UpdateSubConnState: 0x4000530ab0, {TRANSIENT_FAILURE connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:10009: connect: connection refused"}
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.018 [DBG] GRPC: [core] Channel Connectivity change to TRANSIENT_FAILURE
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.018 [DBG] GRPC: [core] Channel Connectivity change to SHUTDOWN
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.018 [DBG] GRPC: [core] Subchannel Connectivity change to SHUTDOWN
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.018 [ERR] LITD: Could not start subservers: error subscribing to lnd wallet state: lnd version incompatible, need at least v0.13.0-beta, got error on state subscription: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:10009: connect: connection refused"
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.018 [DBG] GRPC: [core] Channel Connectivity change to SHUTDOWN
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.018 [DBG] GRPC: [core] Subchannel Connectivity change to SHUTDOWN
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.019 [INF] SGNL: Received shutdown request.
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.019 [INF] SGNL: Shutting down...
Apr 25 20:23:37 raspberrypi torsocks[11945]: 2023-04-25 20:23:37.019 [INF] SGNL: Gracefully shutting down.
Apr 25 20:23:37 raspberrypi torsocks[11945]: error subscribing to lnd wallet state: lnd version incompatible, need at least v0.13.0-beta, got error on state subscription: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:10009: connect: connection refused"
Apr 25 20:23:37 raspberrypi systemd[1]: litd.service: Main process exited, code=exited, status=1/FAILURE
Apr 25 20:23:37 raspberrypi systemd[1]: litd.service: Failed with result 'exit-code'.

*** LAST 20 BTCPayServer LOGS ***
sudo journalctl -u btcpayserver -b --no-pager -n20
-- Journal begins at Mon 2023-04-24 05:39:35 BST, ends at Tue 2023-04-25 20:24:21 BST. --
Apr 25 20:22:34 raspberrypi systemd[1]: Dependency failed for BtcPayServer daemon.
Apr 25 20:22:34 raspberrypi systemd[1]: btcpayserver.service: Job btcpayserver.service/start failed with result 'dependency'.

- BTC-RPC-Explorer is OFF by config
- LNbits is OFF by config

*** LAST 20 Thunderhub LOGS ***
sudo journalctl -u thunderhub -b --no-pager -n20
-- Journal begins at Mon 2023-04-24 05:39:35 BST, ends at Tue 2023-04-25 20:24:21 BST. --
Apr 25 20:23:41 raspberrypi npm[10339]:   err: [ 503, 'FailedToConnectToDaemon' ],
Apr 25 20:23:41 raspberrypi npm[10339]:   level: 'error',
Apr 25 20:23:41 raspberrypi npm[10339]:   message: 'Error connecting to node',
Apr 25 20:23:41 raspberrypi npm[10339]:   timestamp: '2023-04-25T19:23:41.154Z'
Apr 25 20:23:41 raspberrypi npm[10339]: }
Apr 25 20:23:41 raspberrypi npm[10339]: {
Apr 25 20:23:41 raspberrypi npm[10339]:   message: 'Unable to connect to any node.',
Apr 25 20:23:41 raspberrypi npm[10339]:   level: 'error',
Apr 25 20:23:41 raspberrypi npm[10339]:   timestamp: '2023-04-25T19:23:41.156Z'
Apr 25 20:23:41 raspberrypi npm[10339]: }
Apr 25 20:23:41 raspberrypi npm[10339]: {
Apr 25 20:23:41 raspberrypi npm[10339]:   message: 'UnableToConnectToAnyNode',
Apr 25 20:23:41 raspberrypi npm[10339]:   level: 'error',
Apr 25 20:23:41 raspberrypi npm[10339]:   timestamp: '2023-04-25T19:23:41.159Z'
Apr 25 20:23:41 raspberrypi npm[10339]: }
Apr 25 20:23:41 raspberrypi npm[10339]: {
Apr 25 20:23:41 raspberrypi npm[10339]:   level: 'error',
Apr 25 20:23:41 raspberrypi npm[10339]:   message: 'Initiating subscriptions failed: ',
Apr 25 20:23:41 raspberrypi npm[10339]:   timestamp: '2023-04-25T19:23:41.160Z'
Apr 25 20:23:41 raspberrypi npm[10339]: }

- SPECTER is OFF by config
- SPHINX is OFF by config

*** MOUNTED DRIVES ***
Filesystem     Type      Size  Used Avail Use% Mounted on
/dev/root      ext4       29G   21G  7.5G  74% /
devtmpfs       devtmpfs  1.9G     0  1.9G   0% /dev
tmpfs          tmpfs     1.9G   16K  1.9G   1% /dev/shm
tmpfs          tmpfs     769M  1.1M  768M   1% /run
tmpfs          tmpfs     5.0M     0  5.0M   0% /run/lock
tmpfs          tmpfs      32M     0   32M   0% /var/cache/raspiblitz
/dev/mmcblk0p1 vfat      253M   31M  222M  13% /boot
/dev/sdb1      ext4      916G  591G  279G  68% /mnt/hdd
tmpfs          tmpfs     385M   16K  385M   1% /run/user/1000
tmpfs          tmpfs     385M   16K  385M   1% /run/user/1001

*** DATADRIVE ***
# RASPIBLITZ DATA DRIVE Status

# BASICS
isMounted=1
isBTRFS=0
hddRaspiData=1
hddRaspiVersion='1.8.0'
isSSD=0
datadisk='sdb'
datapartition='sdb1'
hddCandidate='sdb'
hddPartitionCandidate='sdb1'
hddBlocksBitcoin=1
hddBytes=1000203091968
hddGigaBytes=931
hddUsedInfo='591G (68%)'
hddDataFreeBytes=298823053312
hddDataFreeKB=291819388
hddDataFreeGB=278
hddAdapterUSB='2537:1068'
hddAdapterUSAP=0

# RAID
isRaid=0
raidCandidate[0]='sda 14 GB Verbatim'
raidCandidate[1]='zram0 4 GB zram0'
raidCandidates=2

# SWAP
isSwapExternal=1
SwapExternalPath='/mnt/hdd/swapfile'

*** NETWORK ***
localip=192.168.178.27
localiprange=192.168.178.0/24
dhcp=1
network_device=wlan0

*** HARDWARE TEST RESULTS ***
UndervoltageReports in Logs: 0

*** SYSTEM CACHE STATUS ***
system_ram_mb="3844"
system_count_undervoltage="0"
system_count_start_tui="0"
system_count_start_lightning="0"
system_init_time="1682450508"
system_up="190"
system_temp_celsius="53"
system_ram_gb="4"
system_temp_fahrenheit="127"
system_vm_vagrant="0"
system_cpu_load=" 2.61, 1.38, 0.55"
system_ups_status="OFF"
system_count_longscan="0"
system_ram_available_mb="2755"
system_ups_battery=""
system_count_start_blockchain="5"
system_board="rp4"
ln_default_error_short=""
ln_default_locked="0"
ln_default_version="0.15.5-beta"
ln_default_online="0"
ln_default_sync_initial_done="1"
ln_default_ready="0"
ln_default_error_full=""
ln_default_running="0"
ln_default_activated="1"
btc_default_sync_initial_done="1"
btc_default_activated="1"
btc_default_sync_percentage="0"
btc_default_ready="0"
btc_default_running="0"
btc_default_online="0"
btc_default_error_full=""
btc_default_peers="0"
btc_default_port=""
btc_default_version="v23.0.0"
btc_default_error_short=""

*** LOGFILES ***
Archived and active journals take up 304.0M in the file system.
889M	/var/log

*** OPTION: SHARE THIS DEBUG OUTPUT ***
An easy way to share this debug output on GitHub or on a support chat
Use the following command and share the resulting link using termbin.com service and tor proxy:
 debug -l
If tor is failing and you don't mind leaking your ip address to the termbin service, use without tor:
 debug -l -n

Press ENTER to return to main menu.

**Comment by KanutoTakumi** [comment, marked as answer]
Ok, this is embarrassing, I should have read the log better.

just delete the old peer.dat and restart.

`sudo mv /mnt/hdd/bitcoin/peers.dat /mnt/hdd/bitcoin/peers.dat.old`

and

`sudo reboot`

sorry for bothering, ser

**Comment by rootzoll** [comment]
Thanks for reporting anyway - lets keep this issue open as an example for the peer.dat error message to be part of the future to be build auto errorn detection:

```
2023-04-25T19:22:39Z Error: Invalid or corrupt peers.dat (Checksum mismatch, data corrupted). If you believe this is a bug, please report it to https://github.com/bitcoin/bitcoin/issues. As a workaround, you can move the file ("/mnt/hdd/bitcoin/peers.dat") out of the way (rename, move, or delete) to have a new one created on the next start.
```
